x-pack/plugins/ingest_pipelines/common/constants.ts

```diff
@@ -9,9 +9,9 @@ const basicLicense: LicenseType = 'basic';
 
 export const PLUGIN_ID = 'ingest_pipelines';
 
-export const PLUGIN_MIN_LICENSE_TYPE = basicLicense;
+export const MANAGEMENT_APP_ID = 'management';
```

Comment:
This seems like something the `management` plugin should define inside a `common/constants.ts` file and export in https://github.com/elastic/kibana/blob/master/src/plugins/management/public/index.ts. Since the `management` plugin is already a dependency of this plugin, it seems like it would be straightforward to make that change and consume the constant wherever we need it like this:

```js
import { MANAGEMENT_APP_ID } from 'src/plugins/management/public';
```

The only slightly invasive change would be you'd have to update the `management` plugin to consume this constant on this line: https://github.com/elastic/kibana/blob/master/src/plugins/management/public/plugin.ts#L75

Reply:
Yeah, I was looking at the management app for an exported constant but couldn't find any. So I added an export, do you know maybe who would be a code owner for this plugin? @cjcenizal

Reply:
That would be the Kibana App Arch team.